### PR TITLE
Euen.failing branch

### PR DESCRIPTION
**Pull request by Euen**

### CODE REVIEW

**Comment by gadget-tester** [comment]
According to **Dialyzer**:
> 
dialyzer: No such file, directory or application: "test/"

File: src/erlang_repo_sup.erl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+-module(erlang_repo_sup).
+-behaviour(supervisor).
+
+-export([start_link/0, init/1]).
+
+-spec start_link() -> [boolean()].
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl]: According to **Dialyzer**:
>  Invalid type specification for function erlang_repo_sup:start_link/0. The success typing is () -> 'ignore' | {'error',_} | {'ok',pid()}

File: src/erlang_repo_sup.erl [hunk 2]
```diff
@@ -0,0 +1,11 @@
+-module(erlang_repo_sup).
+-behaviour(supervisor).
+
+-export([start_link/0, init/1]).
+
+start_link() ->
+    supervisor:start_link({local, ?MODULE}, ?MODULE, []).
+
+init([]) ->
+	Procs = [],
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl]: According to **Elvis**:
> Line 10 has a tab at column 0.

File: src/erlang_repo_sup.erl [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	{ok, {{one_for_one, 1, 5}, Procs}}.
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl]: According to **Elvis**:
> Line 11 has a tab at column 0.

**Comment by Euen** [comment]


File: src/erlang_repo_sup.erl (hunk 3 above)
Comment (gadget-tester) [inline src/erlang_repo_sup.erl:11]: According to **Elvis**:

> Line 11 has a tab at column 0.

File: src/erlang_repo_sup.erl (hunk 2 above)
Comment (gadget-tester) [inline src/erlang_repo_sup.erl:10]: According to **Elvis**:

> Line 10 has a tab at column 0.

**Pull request closed without merging**